<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1947</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>1671</width>
      <height>621</height>
     </rect>
    </property>
    <property name="text">
     <string>hhhh</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>910</y>
      <width>392</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbCamStart">
       <property name="text">
        <string>Kamera starten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCamStop">
       <property name="text">
        <string>Kamera stoppen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCamCal">
       <property name="text">
        <string>Kalibrierung starten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCamCalConfirm">
       <property name="text">
        <string>Best√§tigen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1947</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDer_BESTE_Eyetracker">
    <property name="title">
     <string>Der BESTE Eyetracker</string>
    </property>
   </widget>
   <addaction name="menuDer_BESTE_Eyetracker"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
